packages/react-components/src/components/chart/utils/getInfo.ts

```diff
@@ -104,11 +106,13 @@ export const calculateTrendCursorsSeriesMakers = (
       value = delta + valueMin;
     }
 
+    // TODO: precision to be decided
```

Comment:
how will this be decided? do we have an item in Asana to remind ourselves to get back to this?

Reply:
how, I am not entirely sure and hence have created this task already https://app.asana.com/0/1204920453792313/1205045927987035/f